src/components/Reports/GDPR/SubscriberGdpr.js

```diff
@@ -17,6 +17,19 @@ const SubscriberGdpr = ({ location, dependencies: { dopplerApiClient } }) => {
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
   const [state, setState] = useState({ loading: true });
 
+  const Breadcrumb = () => (
+    <nav className="dp-breadcrumb">
+      <ul>
+        <li>
+          <a href={_('subscriber_history.subscriber_breadcrumb_url')}>
+            {_('subscriber_history.subscriber_breadcrumb')}
+          </a>
+        </li>
+        <li>{_('subscriber_gdpr.gpdr_state_breadcrumb')}</li>
+      </ul>
+    </nav>
+  );
```

Comment:
Isn't any way to put this in one place and export it?

Reply:
i dont know if we can do a general component, because the texts and links changes. What you propose?

Reply:
Could't you pass that information by parameters? Like HeaderNav element for example

Reply:
Sorry, I have no enough context, @lucianaDilizio what is the repeated structure?

Reply:
I think that the breadcrumb is something common that we probably use in other places, so I just asked if there are any way to reuse this code for futures cases.

Reply:
yes maybe after we can think how can create reusable component